ghost/core/core/frontend/src/ghost-stats/ghost-stats.js

```
@@ -0,0 +1,297 @@
+      return null
+    }
+
+    let item = null;
+
+    try {
+      item = JSON.parse(serializedItem)
+    } catch (error) {
+      return null
+    }
+
+    if(typeof item !== 'object' || item === null) {
```

ESLint

[error] Expected indentation of 8 spaces but found 4.

(indent)